packages/rocketchat-ui/client/views/app/createChannel.html

```diff
@@ -1,90 +1,98 @@
 <template name="createChannel">
 	<section class="create-channel">
 		{{#each roomType in roomTypesBeforeStandard}}
-			{{> Template.dynamic template=roomType.creationTemplate }}
+			<div class="room-type-creation">
```

Comment:
@ruKurz I wrapped all custom templates into a `div`. This shall allow to visualize them as tabs, I hope